torchdynamo/variables/builder.py

```diff
@@ -164,19 +166,39 @@ def _wrap(self, value):
             guards = self.make_guards(GuardBuilder.EQUALS_MATCH)
             return RangeVariable(value=value, guards=guards)
         elif istype(value, (dict, collections.OrderedDict)) and all(
-            map(ConstantVariable.is_literal, value.keys())
+            map(
+                lambda k: ConstantVariable.is_literal(k)
```

Comment:
use your new function - `is_valid_key`?

Reply:
There's some subtlety here, when creating a dict from an already created dictionary the keys are not TensorVariables yet, they're raw python values.